Setup/docker-compose_dev.yml

```diff
@@ -140,9 +143,12 @@ services:
         - "CONSUL_BIND_INTERFACE=eth0"
         - "NUMBER_OF_EXPECTED=3" # works for single and multi node
         - "LOGSTASH_FIELDS=service=consulserver"
+        - "DATA_DIR={{.Service.Name}}{{.Task.Slot}}" #single node don't use this for multinode
+    volumes:
+      - consul:/consul/data
 
   consul:
-      image: securebrowsing/shield-configuration:latest
+      image: securebrowsing/shield-consul-agent:latest
```

Comment:
I agree the new name is better, 
what will happen on upgrade? i.e. a system is deployed with shield-configuration component and we deploy a new system with consul-agent
I assume this requires system restart
Please test that we will do the right thing

Reply:
I not think so...
Because this is new service... And all should start automatically with new service....

Reply:
Will test this...